services/docker/docker-version.service.js

```diff
@@ -0,0 +1,141 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { nonNegativeInteger } = require('../validators')
+const { latest, renderVersionBadge } = require('../version')
+const {
+  buildDockerUrl,
+  getDockerHubUser,
+  getMultiPageData,
+  getDigestSemVerMatches,
+} = require('./docker-helpers')
+const { BaseJsonService } = require('..')
+const { NotFound } = require('..')
+
+const buildSchema = Joi.object({
+  count: nonNegativeInteger.required(),
+  results: Joi.array()
+    .items(
+      Joi.object({
+        name: Joi.string().required(),
+        images: Joi.array().items(
+          Joi.object({
+            digest: Joi.string(),
+            architecture: Joi.string().required(),
+          })
+        ),
+      }).required()
+    )
+    .required(),
+}).required()
+
+const queryParamSchema = Joi.object({
+  sort: Joi.string()
+    .valid('date', 'semver')
+    .default('date'),
+}).required()
+
+module.exports = class DockerVersion extends BaseJsonService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return { ...buildDockerUrl('version', true), queryParamSchema }
```

Comment:
I missed this before, but we want to use `v` for the badge route prefix for version badges

https://github.com/badges/shields/blob/master/doc/badge-urls.md

```suggestion
    return { ...buildDockerUrl('v', true), queryParamSchema }
```

Reply:
No problems here.